frontend/src/App.js

```diff
@@ -13,13 +18,16 @@ export default function App() {
       <Router>
         <Switch>
           <Route path="/login" component={Login} />
-          <ProtectedRoute exact path="/results" component={Results} />
+          <ProtectedRoute path="/results" component={Results} />
           <ProtectedRoute path="/settings" component={Settings} />
           <ProtectedRoute
             path="/admin/registration"
             component={AdminRegistration}
           />
           <Route path="/user/registration" component={UserRegistration} />
+          <Route path="/">
+            <Redirect to="/login" />
+          </Route>
```

Comment:
passiert das nicht automatisch wenn du keinen token hast?

Reply:
Bei "/" hat mir kein "/login" gerendert, nach "Redirect to" tut das :)